test/integration/dashboard/inspec/dashboard_spec.rb

```diff
@@ -2,7 +2,7 @@
   describe port(p) do
     it { should be_listening }
     its('protocols') { should include 'tcp' }
-    its('addresses') { should include '0.0.0.0' }
+    its('addresses') { should include '::' }
```

Comment:
Should this be looking for both ipv4 and ipv6?

Reply:
IIRC I had an issue with this test and this was the work around. I'll take another look at it and confirm.

Reply:
Yes this was causing an error for some reason:
```
     ×  addresses is expected to include "0.0.0.0"
     expected ["::"] to include "0.0.0.0"
```
When I got into the system I see the following with `ss`.
```
[::]:80
```
Operationally, everything works fine so I'd like to keep this test as-is.